.github/workflows/pr-build.yaml

```diff
@@ -0,0 +1,237 @@
+          
+        - name: Copy custom settings.xml
+          run: |
+              cp /home/runner/work/secrets/settings.xml /home/runner/work/gpg/settings.xml
+        
+        - name:  Pull and run the docker image
+          run: |
+           docker run --rm -v ${{github.workspace}}:/var/root/ ghcr.io/${{env.NAMESPACE}}/galasabld-amd64:${{env.IMAGE_TAG}} template --releaseMetadata var/root/framework/release.yaml --releaseMetadata /var/root/extensions/release.yaml --releaseMetadata /var/root/managers/release.yaml --releaseMetadata /var/root/obr/release.yaml --template /var/root/obr/galasa-bom/pom.template --output /var/root/obr/galasa-bom/pom.xml --bom
+           
+        - name: Display pom.xml 
+          run: |
+           cat ${{github.workspace}}/obr/galasa-bom/pom.xml
+          
+        
+        - name: Build Obr using maven in directory galasa-bom
+          run: |
+           mvn -f ${{github.workspace}}/obr/galasa-bom/pom.xml deploy -Dgpg.skip=true \
+           -Dgalasa.source.repo=https://development.galasa.dev/gh/maven-repo/managers \
+           -Dgalasa.central.repo=https://repo.maven.apache.org/maven2/ \
+           -Dgalasa.release.repo=file:${{ github.workspace }}/obr/repo \
+           --batch-mode --errors --fail-at-end \
+           --settings  /home/runner/work/gpg/settings.xml
+        
+        - name:  Pull and run the docker image
+          run: |
+           docker run --rm -v ${{github.workspace}}:/var/root/ ghcr.io/${{env.NAMESPACE}}/galasabld-amd64:${{env.IMAGE_TAG}} template --releaseMetadata var/root/framework/release.yaml --releaseMetadata /var/root/extensions/release.yaml --releaseMetadata /var/root/managers/release.yaml --releaseMetadata /var/root/obr/release.yaml --template /var/root/obr/dev.galasa.uber.obr/pom.template --output /var/root/obr/dev.galasa.uber.obr/pom.xml --obr
+            
+        - name: Display pom.xml 
+          run: |
+           cat ${{github.workspace}}/obr/dev.galasa.uber.obr/pom.xml
+         
+        - name: Build Obr using maven in directory dev.galasa.uber.obr
+          run: |
+            mvn -f ${{github.workspace}}/obr/dev.galasa.uber.obr/pom.xml deploy -Dgpg.skip=true \
+            -Dgalasa.source.repo=https://development.galasa.dev/main/maven-repo/managers \
+            -Dgalasa.central.repo=https://repo.maven.apache.org/maven2/ \
+            -Dgalasa.release.repo=file:${{ github.workspace }}/obr/repo \
+            --batch-mode --errors --fail-at-end \
+            --settings  /home/runner/work/gpg/settings.xml
+        
+        - name: Build OBR image for testing
+          uses: docker/build-push-action@v5
+          with:
+            context: .
+            file: ${{github.workspace}}/obr/dockerfiles/dockerfile.obr
+            load: true
+            tags: obr:test
+            build-args: |
+              dockerRepository=${{env.REGISTRY}}
+              tag= ${{env.IMAGE_TAG}}

```

Comment:
Can we remove the space here as I'm not sure if this will cause an issue? `tag=${{env.IMAGE_TAG}}`